aldryn_forms/models.py

```diff
@@ -17,6 +19,7 @@
 from .helpers import is_form_element
 
 
+USES_TREEBEARD = LooseVersion(cms.__version__) >= LooseVersion('3.1')
```

Comment:
I think this should instead actually test for the usage of Treebeard. You're only testing for a version number here, but what you really need, is to know whether treebeard or mptt is in use.